info.md

```diff
@@ -1,22 +1,21 @@
 ### Security Qualitative Metrics Information
-* [Flagship Project(https://img.shields.io/badge/owasp-flagship-blue.svg)](https://owasp.org/projects/)
-* [Type of Project](#)
```

Comment:
This link was invalid

Reply:
I couldn't find a proper link for a flagship svg file. IF you provide a lkink I can update it.